sources/world/admin_divisions_v1.hjson

```diff
@@ -0,0 +1,171 @@
+{
+    versions: '1'
+    type: http
+    production: true
+    wikidata: Q56061
+    note: 'Administrative divisions of countries collected from Natural Earth and OpenStreetMap'
+    data: ''
+    attribution: [
+      {
+        label: {
+          en: Made with NaturalEarth
+        }
+        url: {
+          en: http://www.naturalearthdata.com/about/terms-of-use
+        }
+      }
+      {
+        label: {
+          en: © OpenStreetMap contributors
+        }
+        url: {
+          en: https://www.openstreetmap.org/copyright
+        }
+      }
+      {
+        label: {
+          en: Elastic Maps Service
+        }
+        url: {
+          en: https://www.elastic.co/elastic-maps-service
+        }
+      }
+    ]
+    fieldMapping: [
+        {
+          type: id
+          name: iso_3166_2
+          desc: ISO-3166-2 identifier
+        }
+        {
+          type: id
+          name: iso2
+          desc: Country code
```

Comment:
Country code is the first id-field we're introducing in EMS that is _not_ a unique identifier.

The joining-code in Kibana Maps works fine enough. 

e.g. join on country-code. It captures all the individual regions.

![image](https://user-images.githubusercontent.com/1833023/86930360-ee48cd80-c104-11ea-8392-8436aca1f07a.png)


![image](https://user-images.githubusercontent.com/1833023/86930314-e0934800-c104-11ea-9f1c-794c911f948c.png)

The logic also works for region maps.

![image](https://user-images.githubusercontent.com/1833023/86930680-42ec4880-c105-11ea-94c7-ff50a05856ff.png)

There's some edge-case, e.g. the display shows internal borders.


I guess the overall question is. Do we want to make this a new semantic of the id-field? ie. a field that uniquely identifies some entity, but it can be both a primary key (ie. the region-code) _or_ a foreign key (ie. the country-code).

Reply:
^ this behavior of the inner-join code is not unit-tested in Kibana (all tests expect a primary key in the left-hand side "table").

Reply:
I don't want to make broad changes here. I've removed the country code in the latest update.